include/spi.h

```diff
@@ -72,6 +72,13 @@ typedef enum spi_mode {
 	SPI_MODE_3 = (SPI_CPOL | SPI_CPHA)
 } spi_mode;
 
+/**
+ * @struct spi_platform_ops
+ * @brief Structure holding SPI function pointers that point to the platform
+ * specific function
+ */
+struct spi_platform_ops ;
```

Comment:
Shouldn't the name be spi_ops only? It is not platform dependent.
And instead of xil_ops and altera_ops have only one name: platform_ops.

Reply:
The `spi_platform_ops` holds platform specific information therefore the naming explicit enough. Moreover, the naming was already discussed in the meeting and sent via email.